Unity 2019.3.7f1 upgrade (#1336)
spatialos/gdk-for-unity: CHANGELOG.md

## Unreleased

### Breaking Changes

> - The GDK for Unity now supports Unity 2019.3. You will need to upgrade your Unity project. [#1272](https://github.com/spatialos/gdk-for-unity/pull/1272)

Reviewer: ```suggestion
- The minimum version of Unity supported by the GDK is now 2019.3. You will need to upgrade your Unity project. [#1272](https://github.com/spatialos/gdk-for-unity/pull/1272)
```

Reviewer: makes it clearer why they have to upgrade?

Author: Ye

> - The minimum version of Unity supported by the GDK is now 2019.3. You will need to upgrade your Unity project. [#1272](https://github.com/spatialos/gdk-for-unity/pull/1272)

Author: ```suggestion
- The minimum version of Unity supported by the GDK is now 2019.3. You will need to upgrade your Unity project. [#1336](https://github.com/spatialos/gdk-for-unity/pull/1336)
```

Reviewer: haha

> - The minimum version of Unity supported by the GDK is now 2019.3. You will need to upgrade your Unity project. [#1336](https://github.com/spatialos/gdk-for-unity/pull/1336)

Reviewer 2: ```suggestion
- The minimum version of Unity that the GDK supports is now 2019.3. You need to upgrade your Unity project. [#1336](https://github.com/spatialos/gdk-for-unity/pull/1336)
```
